src/App.tsx

```diff
@@ -1,18 +1,33 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useState } from 'react';
 import './App.css';
-import ToDoList from './ToDoList';
+import ToDoList from './components/ToDoList/ToDoList';
 
-function App() {
+export interface ITodoList {
+  index: number;
+  title: string;
+  isCompleted: boolean;
+}
+export default function App() {
+  const storedTodos = JSON.parse(localStorage.getItem('todos') || 'null');
+  const [todos, setTodos] = useState<ITodoList[]>(storedTodos.length === 0 ? [] : storedTodos)
```

Comment:
```suggestion
 const [todos, setTodos] = useState<ITodoList[] | null>(JSON.parse(localStorage.getItem('todos')))
```

Reply:
corrected to: 
const [todos, setTodos] = useState<ITodoList[]>(JSON.parse(localStorage.getItem("todos") || "[]"))